### PR TITLE
Styling list view

### LINKED ISSUE #51: Update `List.js` input and verbiage to reflect Figma.
Update the Header and input design to resemble our [Figma](https://www.figma.com/file/Gd4a2z8c5iNG6b3ZWs36g4/Smart-Shopping-List?node-id=0%3A1) design.

`List.js` header and input
list-h1-input-figma

### PR DESCRIPTION
**Pull request by apdsrocha**
## Description

This adds a new color to our Tailwind config file, and adds classes to style our list view to better reflect Figma.

## Related Issue

closes #51 

### After

mobile-01
desktop-01

## Notes:

I'll open an issue so that we can address our empty state view. 
Screen Shot 2021-05-29 at 2 08 28 PM

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = {
         'caribbean-green': '#06D6A0',
         'blue-ncs': '#118AB2',
         'midnight-green': '#073B4C',
+        teal: '#CCFBF1',
```
Comment (jssckbl) [inline tailwind.config.js]: Does `teal` require quotes around it, at least for consistency? Probably very minor, but it is the only thing I could notice that would possibly need to be addressed before merging.

**Review by jssckbl — COMMENTED** [review]
Overall, the `List.js` view is looking like what was designed in Figma! Nice work 😄

File: tailwind.config.js (hunk 1 above)
Comment (apdsrocha) [reply tailwind.config.js]: You're absolutely right! Let me get that fixed :)

File: src/pages/List.js [hunk 2]
```diff
@@ -269,83 +269,110 @@ export default function List({ token }) {
...
-          icon={
+      <h1 className="mt-5 mb-10 text-3xl self-start font-light">
+        Things I'll need
+      </h1>
+      <div
+        class="bg-teal border-t-4 rounded-b text-midnight-green border-caribbean-green px-4 py-3 shadow-md w-full"
+        role="alert"
+      >
+        <div class="flex">
+          <div class="py-1 text-caribbean-green">
```
Comment (jamesncox) [inline src/pages/List.js]: Looks good! A lot like the Figma design!

Just a quick reminder to change `class` to `className`!

File: src/pages/List.js [hunk 3]
```diff
@@ -269,83 +269,110 @@ export default function List({ token }) {
...
             <svg
+              class="fill-current h-6 w-6 text-teal-500 mr-4"
```
Comment (jamesncox) [inline src/pages/List.js]: Same here with `class` to `className`

File: src/pages/List.js [hunk 4]
```diff
@@ -269,83 +269,110 @@ export default function List({ token }) {
...
-              />
+              <path d="M2.93 17.07A10 10 0 1 1 17.07 2.93 10 10 0 0 1 2.93 17.07zm12.73-1.41A8 8 0 1 0 4.34 4.34a8 8 0 0 0 11.32 11.32zM9 11V9h2v6H9v-4zm0-6h2v2H9V5z" />
             </svg>
-          }
-          label="clear input"
-        />
+          </div>
+          <div class="flex items-center justify-around w-full">
+            <p class="font-bold">Here's your token</p>
+            <p class="text-sm">{token}</p>
```
Comment (jamesncox) [inline src/pages/List.js]: You guessed it! `class` ===> `className`

**Review by jamesncox — CHANGES_REQUESTED** [review]
Looks great! Just have to make it "JSX-friendly" with `className`!

File: src/pages/List.js (hunk 2 above)
Comment (apdsrocha) [reply src/pages/List.js]: Will do! 👍

**Comment by jamesncox** [comment]
LOOKS GREAT! PR APPROVED!!

**Pull request merged**